sdk/samples/iot/paho_iot_pnp_sample.c

```diff
@@ -796,36 +698,36 @@ static void handle_command_request(
   }
   else
   {
-    IOT_SAMPLE_LOG_AZ_SPAN("Command not supported:", command_request->name);
+    IOT_SAMPLE_LOG_AZ_SPAN("Command not supported:", command_request->command_name);
     send_command_response(command_request, AZ_IOT_STATUS_NOT_FOUND, command_empty_response_payload);
   }
 }
 
 static void send_command_response(
-    az_iot_hub_client_method_request const* command_request,
+    az_iot_pnp_client_command_request const* command_request,
     az_iot_status status,
     az_span response)
 {
   az_result rc;
 
-  // Get the Methods response topic to publish the command response.
-  char methods_response_topic_buffer[128];
-  rc = az_iot_hub_client_methods_response_get_publish_topic(
-      &hub_client,
+  // Get the command response topic to publish the command response.
```

Comment:
nit: commands plural
here and elsewhere for commands response topic (basing off function name az_iot_pnp_client_commands_response_get_publish_topic)

Reply:
Yea I actually was looking to change the API name to drop the `s` but it aligns with methods. Arg